SECURITY.md

```diff
@@ -0,0 +1,24 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+If you have discovered a security vulnerability in this project, please report it privately. **Do not disclose it as a public issue.**
+This gives us time to work with you to fix the issue before public exposure, reducing the chance that the exploit will be used before a patch is released.
+
+Please submit the bug report using the [security vulnerability report form](https://github.com/unicode-org/icu4x/security/advisories/new) on GitHub.
+
+Please provide the following information in your report:
+
+ * A description of the vulnerability and its impact
+ * How to reproduce the issue
+
+This project is maintained by volunteers on a reasonable-effort basis. As such, we ask that you give us 90 days to work on a fix before public exposure.
+
+
+## Supported Versions
```

Comment:
Suggestion: I'm not sure this section adds a lot of value. I think I'd rather just add a bullet point "impacted versions" in the list of information to include in the report. If you feel the need, say something like "We will review reports impacting any version and will prioritize those impacting recently published versions."

Reply:
It's pretty standard to include _some_ idea of which versions are "security patch supported" in security policies; and GitHub's default template encourages this.